src/main/resources/templates/mail/listOfNotActivatedUsers.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html xmlns:th="http://www.thymeleaf.org" lang="en">
+<head>
+  <title>Not Activated Accounts Created in Last Week</title>
+  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+  <link rel="shortcut icon" th:href="@{|${baseUrl}/favicon.ico|}" />
+  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css" integrity="sha384-JcKb8q3iqJ61gNV9KGb8thSsNjpSL0n8PARn9HuZOnIxN0hoP+VmmDGMN5t9UJ0Z" crossorigin="anonymous">
+</head>
+<body>
+<p>
+  Dear members,
+</p>
+<p>
+  The following OncoKB account(s) have been created and verified in the last week but have not been activated:
```

Comment:
Let's make this email a bit more generic. Let's do last `{duration} day(s)` instead of last week. We need to change the title as well.